proto/asn1/asn1-universal-types/asn1_universal_types.proto

```diff
@@ -0,0 +1,103 @@
+// This proto represents the Universal types
+// found in X509 Certifiactes.
+syntax = "proto2";
+
+import "asn1_pdu.proto";
+
+package asn1_universal_types;
+
+// See X.690 (2015), 8.6.
+message BitString {
+  // BitString is either constructed or primitive (X.690 (2015), 8.6.1).
+  required asn1_pdu.Encoding encoding = 1;
+  required bytes val = 2;
+}
+
+// See X.690 (2015), 8.3.
+message Integer {
+  required bytes val = 2;
+}
+
+// See X.690 (2015), 11.8.
+message UTCTime {
+  required TimeValue0To9 year_tens_digit = 1;
+  required TimeValue0To9 year_ones_digit = 2;
+  required TimeValue0To1 month_tens_digit = 3;
+  required TimeValue0To9 month_ones_digit = 4;
+  required TimeValue0To3 day_tens_digit = 5;
+  required TimeValue0To9 day_ones_digit = 6;
+  required TimeValue0To2 hour_tens_digit = 7;
+  required TimeValue0To9 hour_ones_digit = 8;
+  required TimeValue0To5 minute_tens_digit = 9;
+  required TimeValue0To9 minute_ones_digit = 10;
+  required TimeValue0To5 second_tens_digit = 11;
+  required TimeValue0To9 second_ones_digit = 12;
```

Comment:
Sorry, should have written down more precise notes here during our meeting.

The intent was something like:

```protobuf
enum MonthsValue {
  JANUARY = 1,
  FEBRUARY = 2,
  ...,
  DECEMBER = 12
}
```

and then
```protobuf
enum HoursValue {
  VALUE_01 = 1,
  ...
  VALUE_24 = 2
}
```
etc

However, we also talked about how this could be done simpler, when were discussing https://github.com/google/fuzzing/pull/52/files#r463796989 , and how you could use [`gmtime`](http://www.cplusplus.com/reference/ctime/gmtime/), avoiding the need for reflection entirely.

Reply:
If we use `gmtime`, then the fuzzer won't be able to govern the input. I'm thinking then we should use the proto approach as outlined above.

Reply:
Right, but we covered that on Friday in our meeting and then again today over e-mail that it should be reasonably fine, since our goal is to produce a semblance of "valid DER" here, and the conditionally-encoded `PDU` element for the TimeChoice is sufficient here to fuzz the time function itself.